### DISCUSSION TITLE
When is version 1.0 surfacing ?

### DISCUSSION THREAD

**Discussion by GMas0124816** [AvaloniaUI/Avalonia#8384] [Q&A] [answered]
Being a .NET developer has never been more exciting than these days, finally cross-platform out of the box. Unfortunately the GUI field isn't yet where it should be for professional applications (I know some run on Avalonia). Ask 10 developers what to use : MAUI, UNO or Avalonia and you will have 10 different answers ... ... personally (mostly desktop development), I keep coming back to avalonia, but like a lot of pro's, I'm waiting on the stability of a version 1.0, not on yet another platform supported. If I want to do mobile, I will go with MAUI or UNO. The core of avalonia has always been desktop, so please keep focusing on that and make a version 1.0 !

I'm following these frameworks for over two years now and to be honest, I had expected that avalonia would have reached version 1.0 by now.

Back to my question, when can we expect that version 1.0 will be released ?

(and I know version 0.10 is some sort of version 1.0, but really ...)

Don't get me wrong, I like a lot what is in avalonia and can only complement the team on what they have done so far.

**Comment by maxkatz6** [comment, marked as answer]
Hi!

> and I know version 0.10 is some sort of version 1.0, but really 

Well, that's the answer. 0.10 was the stable version with keeping binary compatibility between minor 0.10.x versions.

> Back to my question, when can we expect that version 1.0 will be released

Partially related to the fact that 0.10 was stable, the answer is - never. 
Instead we will make versioning less confusing by making next major release with version 11.0. 
"0" as a major version number is pretty much meaningless now. First preview is planned in July.

**Reply by GMas0124816** [reply]
Hi maxkatz6, thanks for the quick answer !

That is good news ! Are there any (major) changes in the api between version 0.10 and 11.0 or should it be very easy to upgrade ?

**Reply by kekekeks** [reply]
There are breaking changes here and there, but nothing that changes the general layout or flow of the application. 

I think the most annoying ones are the removal of `T` in `OnPropertyChanged<T>` and some properties being moved from TextBlock to TextElement.

**Reply by GMas0124816** [reply]
OK kekekeks, thanks for the answer.

I will start with this project with 0.10 and upgrade to 11.0 when it is out of preview. I expect there will be some 'conversions guide' by then ?

**Reply by GMas0124816** [reply]
Any idea when it will be out of preview ? Would start the project in 0.10 and upgrade to 11.0 when out of preview.
